--- origsrc/enca-1.9/Makefile.am	2005-11-24 04:21:32.000000000 -0600
+++ src/enca-1.9/Makefile.am	2007-08-30 21:47:44.750000000 -0500
@@ -41,7 +43,7 @@
 test: check
 
 docs:
-	cd $(srcdir)/devel-docs && $(MAKE) docs
+	cd devel-docs && $(MAKE) docs
 
 .PHONY: test docs
 
--- origsrc/enca-1.9/configure.ac	2005-12-18 06:07:46.000000000 -0600
+++ src/enca-1.9/configure.ac	2007-08-30 21:47:44.765625000 -0500
@@ -90,7 +90,7 @@
 done
 
 dnl And for gtk-doc
-gtk_CHECK_GTK_DOC
+GTK_DOC_CHECK
 
 dnl Check for good random number sources
 AC_CHECK_FILES(/dev/random /dev/urandom /dev/srandom /dev/arandom)
--- origsrc/enca-1.9/devel-docs/Makefile.am	2004-05-27 12:17:15.000000000 -0500
+++ src/enca-1.9/devel-docs/Makefile.am	2007-08-30 21:47:44.781250000 -0500
@@ -84,8 +84,8 @@
 	$(DOC_MODULE).intefraces
 
 if ENABLE_GTK_DOC
-if MAINTAINER_MODE
 docs: html-build.stamp
+all: html-build.stamp
 
 #### scan ####
 
@@ -138,7 +138,6 @@
 	cd $(srcdir) && gtkdoc-fixxref --module-dir=html --html-dir=$(HTML_DIR) $(FIXXREF_OPTIONS)
 	touch html-build.stamp
 endif
-endif
 
 ##############
 
--- origsrc/enca-1.9/lib/Makefile.am	2005-11-24 14:11:20.000000000 -0600
+++ src/enca-1.9/lib/Makefile.am	2007-08-30 21:47:44.796875000 -0500
@@ -7,7 +7,7 @@
 # Any iface change    C++: 0:   A
 # Adding ifaces       C:   R:   A++
 # Changing ifaces     C:   R:   0
-libenca_la_LDFLAGS = -version-info 5:1:5
+libenca_la_LDFLAGS = -version-info 5:1:5 -no-undefined
 libenca_la_SOURCES = \
   common.c \
   ctype.c \
--- origsrc/enca-1.9/m4/libiconv.m4	2004-11-17 12:00:28.000000000 -0600
+++ src/enca-1.9/m4/libiconv.m4	2007-08-30 21:47:44.796875000 -0500
@@ -18,7 +18,7 @@
 
 dnl Use standard iconv test
 AM_ICONV
-CONVERTOR_LIBS="$CONVERTOR_LIBS $LIBICONV"
+CONVERTOR_LIBS="$CONVERTOR_LIBS $LTLIBICONV"
 
 dnl Compile iconvcap.c and run it to determine what encodings iconv actually
 dnl knows an under what names. This is not needed with GNU iconv. HAVE_ICONV
